test/e2e-test/suite_test.go

```diff
@@ -218,6 +218,51 @@ var _ = BeforeSuite(func(done Done) {
 	}
 	Expect(k8sClient.Create(context.Background(), &crd)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
 	By("Created a crd for appconfig dependency test")
+
+	crd = crdv1.CustomResourceDefinition{
```

Comment:
Why can't this test re-use the crd for the appconfig dependency test?

Reply:
I'm afraid the Definition（cluster scope CRD） file will conflict, so I create a new one.

Reply:
how do they conflict?

Reply:
using it as traitdefinition, revisionEnabled true/false will affect the test result.